lib/tests/test_local_working_copy.rs

```diff
@@ -601,6 +601,78 @@ fn test_materialize_conflicted_files() {
     4
     >>>>>>> Conflict 1 of 1 ends
     "###);
+
+    // Editing a conflicted file should correctly propagate updates to each of
+    // the conflicting trees.
+    testutils::write_working_copy_file(
+        &workspace_root,
+        file1_path,
+        r##"<<<<<<< Conflict 1 of 1
```

Comment:
I'd use `indoc!` here. Apart from the arguable style improvement, this means that this file will not have any valid conflict markers. So, `jj` will not get confused if one day there's a commit with a conflict in this file.